### PR TITLE
Refactors for RSKIP326.

### PR DESCRIPTION
**Pull request by jeremy-then**
Refactors for RSKIP326.
Removed unnecessary util functions to convert from hash160 to base58 because it is no longer needed with RSKIP326.
Updated test.

### CODE REVIEW

**Review by nathanieliov — COMMENTED** [review]
Update version on package.json

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "nyc mocha --timeout 10000"
   },
   "dependencies": {
-    "@rsksmart/rsk-precompiled-abis": "^4.0.2-IRIS",
+    "@rsksmart/rsk-precompiled-abis": "https://github.com/rsksmart/precompiled-abis#0cf4e50f83b0ef9499608efb46c2fc3f7bb973cd",
```
Comment (kelvinator07) [inline package.json:10]: Is There a Reason Why You Using This Instead of 4.0.2-IRIS or 6.0.0-HOP?

File: index.js [hunk 2]
```diff
@@ -119,13 +119,10 @@ const decodeLogs = (web3Client, tx, bridge, network) => {
                 let value;
                 if (input.indexed) {
                     value = web3Client.eth.abi.decodeParameter(input.type, txLog.topics[topicIndex]);
-                    topicIndex ++;
+                    topicIndex++;
                 } else {
                     value = dataDecoded[input.name];
                 }
-                if (input.name === "btcDestinationAddress") {
```
Comment (marcos-iov) [inline index.js:126]: I think we no longer need the `network` parameter in this method

**Review by marcos-iov — APPROVED** [review]
Nice!

**Pull request merged**